admin/.storybook/preview.js

```diff
@@ -11,7 +13,15 @@ export const parameters = {
   },
 }
 
+const messages = {en: null, fr: null};
+setIntlConfig({
+    locales: ["en", "fr"],
+    defaultLocale: "fr",
```

Comment:
Any reason this is `FR` and the app [this is](url) `EN`?

Reply:
Oh darn, that's a mistake, left over from testing something